splice_machine/src/main/java/com/splicemachine/derby/ddl/DDLWatchRefresher.java

```diff
@@ -214,7 +216,30 @@ private void clearFinishedChanges(Collection<String> children,Collection<DDLWatc
     private void assignDDLDemarcationPoint(DDLChange ddlChange) {
         try {
             TxnView txn = new LazyTxnView(ddlChange.getTxnId(),txnSupplier,exceptionFactory);
-            assert txn.allowsWrites(): "DDLChange "+ddlChange+" does not have a writable transaction";
+            // A full Restore operation overwrite SPLICE_TXN, so the transaction used by the restore
+            // may not be found.  Avoid the assertion to avoid crashing, e.g.,
+            // 2020-08-11 18:28:11,785 (ZooKeeperDDLWatcherRefresher) ERROR [c.s.d.d.AsynchronousDDLWatcher] - Failed to refresh ddl
+            //java.lang.RuntimeException: com.splicemachine.si.api.txn.TransactionMissing: Couldn't resolve transaction with id 81981952
```

Comment:
Can you remove the stack trace from the comment? I think it makes sense to have it on the JIRA but not here (feel free to keep the explanation and a pointer to the JIRA for instance)

Reply:
Done.